Sources/Magnetic.swift

```diff
@@ -28,8 +28,28 @@ open class Magnetic: SKScene {
      Controls whether you can select multiple nodes.
      */
     open var allowsMultipleSelection: Bool = true
+
+    /**
+     Lets the user move individule nodes.
+     **/
+    open var allowSingleNodeMovement: Bool = false
+    
+    /**
+     Lets the user move all of the nodes at once.  On by default.
+     **/
+    open var allowAllNodeMovement: Bool = true
```

Comment:
aren't allowSingleNodeMovement and allowAllNodeMovement mutually exclusive?

Reply:
First, thank you for the feedback.  They could be but it more fun having both on like this:
![2018-02-23 10_48_42](https://user-images.githubusercontent.com/1450431/36611072-2167b5a6-1887-11e8-979a-25be56936fea.gif)

Should update the code comments to be more descriptive?